wvd-templates/wvd-management-ux/deploy/scripts/createWvdMgmtUxAppRegistration.ps1

```diff
@@ -29,12 +29,12 @@ param(
 
 	[Parameter(Mandatory = $true)]
 	[ValidateNotNullOrEmpty()]
-	[string]$SubscriptionId
+	[string]$AzureSubscriptionId
 
 )
 
 # Set the ExecutionPolicy
-Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope LocalMachine -Force -Confirm:$false
```

Comment:
This is the only thing in this script that should remain the same. Please revert everything else back.

Reply:
Perfect, thanks.